code/game/atoms.dm

```diff
@@ -305,8 +306,8 @@
 /atom/proc/ex_act()
 	return
 
-/atom/proc/blob_act()
-	return
+/atom/proc/blob_act(obj/structure/blob/B)
+	SEND_SIGNAL(src, COMSIG_ATOM_BLOB_ACT, B)

```

Comment:
Doesn't this mean every single `blob_act` in the game needs to `..()` 

Also, this would also mean that they would need to have `obj/structure/blob/B` as their arguments.